packages/quantic/force-app/main/default/lwc/quanticSearchBoxInput/__tests__/quanticSearchBoxInput.test.js

```diff
@@ -250,27 +312,147 @@ describe('c-quantic-search-box-input', () => {
             setupEventListeners(element);
             await flushPromises();
 
+            const input = element.shadowRoot.querySelector(
+              textareaValue
+                ? selectors.searchBoxTextArea
+                : selectors.searchBoxInput
+            );
+            expect(input).not.toBeNull();
+
+            await input.focus();
+
             const suggestionsList = element.shadowRoot.querySelector(
               selectors.searchBoxSuggestionsList
             );
             expect(suggestionsList).not.toBeNull();
 
-            const firstSuggestion =
-              suggestionsList.shadowRoot.querySelectorAll('li')[0];
+            const querySuggestionIndex = 0;
+            const firstSuggestion = suggestionsList.shadowRoot.querySelectorAll(
+              selectors.suggestionOption
+            )[querySuggestionIndex];
             expect(firstSuggestion).not.toBeNull();
 
-            firstSuggestion.click();
+            firstSuggestion.dispatchEvent(new CustomEvent('mousedown'));
+            expect(
+              functionsMocks.exampleSelectSuggestion
+            ).toHaveBeenCalledTimes(1);
+
+            /** @type{object} */
```

Comment:
why do you need this?

Reply:
In order to avoid having typing issues when trying to check the payload of the event.
Improved that the type here to make it more precise and more representative about the expected payload.

Reply:
I mean, stating it's a "object" is pretty much the same as disabling ts checking for this object :D 

It's all good though.